deployments/multus-daemonset-thick.yml

```diff
@@ -163,16 +164,19 @@ spec:
           volumeMounts:
             - name: cni
               mountPath: /host/etc/cni/net.d
-            - name: cnibin
```

Comment:
Hey Tomo, why do we remove the bindir, but we keep the other directories? e.g. netns, confdir, etc.

Reply:
In this yaml case (multus-thick with chroot), when the delegate CNI is executed, multus changed the root to host root, with chroot(), hence, we don't need to configure binDir for pod mount namespace. We just configure host directory in this case. Of course, we just use multus-daemon without chroot option, user need to configure.